include/oneapi/dpl/distributed_ranges

```diff
@@ -0,0 +1,28 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_DISTRIBUTED_RANGES
+#define _ONEDPL_DISTRIBUTED_RANGES
+
+#include "oneapi/dpl/internal/common_config.h"
+#include "oneapi/dpl/pstl/onedpl_config.h"
+
+#if __cplusplus > 202002L
+#include "oneapi/dpl/internal/distributed_ranges_impl/sp.hpp"
+#else
```

Comment:
Should `concepts.hpp` also be directly included here - perhaps preceding the other file(s) - to indicate that the concepts are a part of the public API, if for nothing else?

Reply:
Makes sense, done in #1748

Reply:
Applied in https://github.com/oneapi-src/oneDPL/pull/1748, merged, resolving conversation